templates/patterns/date_block/pattern-date-block.html.twig

```diff
@@ -0,0 +1,15 @@
+{#
+/**
+ * @file
+ * Date-block.
+ */
+#}
+{% set attributes = create_attribute() %}
+
+{% include '@oe-bcl/bcl-date-block/date-block.html.twig' with {
+  'day': day,
+  'month': month,
+  'date_time': date_time,
+  'year': year,
+  'attributes': attributes,
```

Comment:
what's the point if it's an empty object ?

Reply:
I'll have a call to explain that. In bcl component date_block there's no initializing of attributes object, so when adding the classes these got lost. So we moved here the initialize of the object so we can pass it to the component and it gets painted correctly.